api/admin/controller/library_settings.py

```diff
@@ -134,8 +135,6 @@ def check_for_missing_settings(self, settings):
             )
 
     def check_input_type(self, settings):
-        # Once there are validations for email address, URL, etc., they'll get called
-        # from here; for now, it's just image type.
```

Comment:
It's OK to remove this comment because it's no longer accurate but as you do this work, you should be moving closer to having a general form validation function that everybody uses.